products/crop_mask_product.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: crop_mask
+description: prediction based s2 l2a gm_tmad, NDVI, LAI, MNDWI, rainfall, slope
+metadata_type: eo3
+
+license: CC-BY-4.0
+
+metadata:
+  product:
+    name: crop_mask
+
+measurements:
+  - name: MASK
```

Comment:
I would prefer lower case here

Reply:
I am refering s2_l2a band "B01" like name.

```
measurements:
  - name: "B01"
    aliases: [band_01, coastal_aerosol]
    units: "1"
    dtype: uint16
    nodata: 0
```

Reply:
I already generating the data with such naming pattern.